handbook/product.md

```diff
@@ -57,8 +57,8 @@ device data in unique ways to their organization.
 ##### How?
 
 - Add integrations for policy and vulnerability automations as well as get closer to parity with
-  other vulnerability management solutions(Miss).
-- Roll up software and vulnerabilities across the entire organization and teams(Hit).
+  other vulnerability management solutions (Miss).
```

Comment:
@noahtalerman, the second half of this sentence is confusing and I'm not sure what it means. Consider rewriting for clarity.

Reply:
@Desmi-Dizney great catch. I broke the confusing sentence into two sentences.

I also updated "get closer to parity" to "take steps toward feature parity."

What do you think?

Reply:
I like it! Thanks for your help.